objects/src/notes/mod.rs

```diff
@@ -19,6 +19,7 @@ pub use vault::NoteVault;
 #[derive(Debug, Eq, PartialEq)]
 pub struct Note {
     script: NoteScript,
+    script_inputs: NoteScriptInputs,
```

Comment:
nit: I'd probably change this to just `inputs`.

Also, the doc comment for the struct needs to be updated.